great_expectations/rule_based_profiler/types/data_assistant_result/data_assistant_result.py

```diff
@@ -97,6 +104,10 @@ class DataAssistantResult(SerializableDictDot):
         "usage_statistics_handler",
     }
 
+    IN_JUPYTER_NOTEBOOK_KEYS = {
+        "execution_time",
+    }
+
```

Comment:
Could we add a comment that let's us know what this is for?

Reply:
@cdkini I can add a comment (though the constant's name accurate describes it too).  Thanks.